resolwe/flow/engines/__init__.py

```diff
@@ -79,25 +83,8 @@ def communicate(self, run_sync=False, verbosity=1):
                         data.save()
                         continue
 
-                    script_template = data.process.run.get('script', '')
-                    inputs = data.input.copy()
-                    hydrate_input_references(inputs, data.process.input_schema)
-                    # hydrate_input_uploads(inputs, data.process.input_schema)
-
-                    info = {}
-                    # info['case_ids'] = data.case_ids
-                    info['data_id'] = data.id
-                    # info['data_path'] = settings.RUNTIME['data_path']
-                    # info['slugs_path'] = settings.RUNTIME['slugs_path']
-                    inputs['proc'] = info  # add script info
-
-                    try:
-                        script = template.Template('{% load resource_filters %}{% load mathfilters %}' +
-                                                   script_template).render(template.Context(inputs))
-                    except template.TemplateSyntaxError as ex:
-                        data.status = Data.STATUS_ERROR
-                        data.process_error.append('Error in process script: {}'.format(ex))
-                        data.save()
+                    script = self.exprengines['dtlbash'].eval(data)
```

Comment:
We have one expression engine now. Just load it. When we add the next one, we can extend the functionality. We will have to think about this more... Let's try to avoid premature optimization.